website/src/pages/privacy.js

```diff
@@ -1,4 +1,4 @@
-import LayoutDefault from "components/layouts/LayoutDefault";
+import LayoutDefault from 'components/layouts/LayoutDefault';
```

Comment:
this would be a good page to use as the sEEd ;)

Reply:
good memory... and good call.